website/content/docs/enterprise/ent-to-ce-downgrades.mdx

```diff
@@ -0,0 +1,173 @@
+---
+layout: docs
+page_title: Enterprise to CE downgrade Process
+description: >-
+  Guide to downgrade from consul enterprise to consul ce
+---
+
+## Introduction
+
+This document describes how to downgrade from Consul Enterprise to Consul community edition (CE).
+
+## Download the CE binary version
+
+First, download the binary for the CE version you want.
+
+<Tabs>
+<Tab heading="Binary">
+
+All current and past versions of the CE and Enterprise releases are
+available on the [HashiCorp releases page](https://releases.hashicorp.com/consul)
+
+</Tab>
+<Tab heading="Docker">
+
+Docker containers are available on [Docker Hub](https://hub.docker.com/r/hashicorp/consul)
+
+</Tab>
+
+</Tabs>
+
+## Prepare for the downgrade to CE
+
+1. Take a snapshot:
+
+    ```
+    consul snapshot save backup.snap
+    ```
+
+1. Inspect the snapshot to ensure that Consul successfully captured it using the following command:
+
+    ```
+    consul snapshot inspect backup.snap
+    ```
+
+    Example output:
+
+    ```
+    ID           2-1182-1542056499724
+    Size         4115
+    Index        1182
+    Term         2
+    Version      1
+    ```
+
+    Inspecting the snapshot ensures that you have a safe fallback option in case something goes wrong. Store
```

Comment:
```suggestion

   ```
   consul snapshot inspect backup.snap
   ```

   Example output:

   ```
   ID           2-1182-1542056499724
   Size         4115
   Index        1182
   Term         2
   Version      1
   ```

   Inspecting the snapshot ensures that you have a safe fallback option in case something goes wrong. Store
```